prov/tcp/src/tcpx.h

```diff
@@ -265,6 +276,10 @@ void tcpx_xfer_entry_release(struct tcpx_cq *tcpx_cq,
 
 void tcpx_progress(struct util_ep *util_ep);
 void tcpx_ep_progress(struct tcpx_ep *ep);
+void tcpx_ep_set_endian_callbacks(struct tcpx_ep *ep, bool endianess);
```

Comment:
endianess may be better named as endian_match.  Though, we can see if having a separate call is needed